clarity_epp.py

```diff
@@ -183,13 +195,18 @@ def placement_complete_step(args):
     subparser = parser.add_subparsers()
 
     output_parser = argparse.ArgumentParser(add_help=False)
-    output_parser.add_argument('-o', '--output_file',  nargs='?', type=argparse.FileType('w'), default=sys.stdout, help='Output file path (default=stdout)')
+    output_parser.add_argument(
+        '-o', '--output_file',  nargs='?', type=argparse.FileType('w'), default=sys.stdout,
```

Comment:
nargs=? --> (a single value, which can be optional)
shouldn't it be nargs=1 ? (3: 3 values, can be any number you want)
or is the output optional here?

Reply:
Yes output (file) is optional -> default is stdout.

see https://docs.python.org/3/library/argparse.html#nargs